New package: GalerkinToolkit v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: GalerkinToolkit
- Repository: https://github.com/fverdugo/GalerkinToolkit.jl
- Created by: @fverdugo
- Version: v0.1.0
- Commit: afb6ae1908476c35fd6f4f404a42ca1d7da23f12
- Reviewed by: @fverdugo
- Reference: https://github.com/fverdugo/GalerkinToolkit.jl/commit/afb6ae1908476c35fd6f4f404a42ca1d7da23f12#commitcomment-143865795
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6fb8475947b6546c736d11394616d2376073962f75767de20c7f50875695f234bed7524afbaee727bfd959a66e6e3054704496f9d992ea29d5bd92db0767e86a9d13489e65493920a1c762d36e792c0c9a9ef14c02996c45c0651d3f190dc9341c4ff17db2432e0ddd3ab0bc1c32d83955191460be2b8827c84e111b9d2ea6ff835d5c4dacc76ba7620b5d443bc941373c6c62e721b3a2c19fbaf534ffbbf397b9cf4de1cd65681c35cbc1daa9c4d43f5 -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README

JuliaRegistrator committed:
New package: GalerkinToolkit v0.1.0

UUID: 5e3ba9c4-dd81-444d-b69a-0e7bd7bf60a4
Repo: https://github.com/fverdugo/GalerkinToolkit.jl.git
Tree: dc5ef884a6ba73b36a4ff89bd8dec80ee2d41b36

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

fverdugo commented:
@goerz Thanks for your useful comment. I have improved the README significantly. [noblock]

goerz commented:
Great, thanks! All but one of the links from the README to the documentation are broken, but otherwise, that looks good! [noblock]

fverdugo commented:
Thanks @goerz for reporting the issues with the links.

 I have fixed them. (The link to the stable documentation will work once there is a first release)

[noblock]
